### PR TITLE
Adding some peer reviewer process

### PR DESCRIPTION
**Pull request by cicorias**
some steps for the Peer Reviewer to follow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,4 +69,20 @@ When you have finished a draft of your case study that you feel is ready for pee
 
 It is probably a good idea to notify potential reviewers in your pull request. Be sure to @mention (at least) two reviewers: (1) a domain expert in the topic you cover, and (2) an engineer who is not familiar with the topic.
 
+Peer reviewers are recomended to add line level comments when they can. This is done on the pull request (PR) that the author has submitted. 
+
+### Identifying what needs Peer Review ###
```
Comment (felixrieseberg) [inline README.md]: Can you stick with the GitHub style and remove the ### at the end of the headers?

**Comment by felixrieseberg** [comment]
Good stuff! Would you mind squashing your commits down? The Git tutorial has a step by step if it's your first time.

**Comment by felixrieseberg** [comment]
Nevermind, already happened :) Thanks!

File: README.md (hunk 1 above)
Comment (cicorias) [inline README.md]: ok, done.

**Pull request merged**